pkg/generator/steps/layout/tpl-new/py-services/_service_/generated/openapi/app.py.tpl

```diff
@@ -0,0 +1,123 @@
+{{.TplHeader}}
+
+import os
+import connexion
+import aiohttp_cors
+import uuid
+import time
+import traceback
+
+from aiohttp import web
+from libraries.generated.metrics.request_metrics import RequestInfo
+from {{.ServiceName}}.generated.core.request_context import MifyRequestContextBuilder
+
+
+async def request_context(request: web.Request):
+    service_ctx = request.config_dict['service_context']
+    request_ctx_builder = MifyRequestContextBuilder(service_ctx)
+    request_ctx_builder.set_request_id(str(uuid.uuid4()))
+    request_ctx_builder.set_url_path(request.path)
+    request_ctx_builder.set_protocol(request.url.scheme)
+    request.app['request_context_builder'] = request_ctx_builder
+    return request_ctx_builder
+
+
+def trim_data(data, limit):
+    if data is None:
+        return '(null)', 0
+    size = len(data)
+    if size > limit:
+        data = data[:limit] + '...'
+    return data, size
+
+
+async def log_request(ctx: MifyRequestContextBuilder, request: web.Request):
+    body, orig_size = trim_data(await request.text(), limit=1024)
+    ctx.logger.bind(request_body=body, size=orig_size).info("started processing request")
```

Comment:
Logging the request body is bad practice. It could contain user data or even some tokens.

Reply:
This is enabled for now for debugging purposes, it will be disabled by default later and we'll give more options to control the logging information to users.